index.html

```diff
@@ -0,0 +1,78 @@
+            <a href="#contact" class="contact">Contact</a>
+    </section>
+    </div>
+</nav>
+<div id="home" class="hero-image">
+    <div class="hero-text">
+        <h1>Sobia Furqan</h1>
+        <p>Software Developer</p>
+        <button> <a href="#contact">Contact</a></button>
+    </div>
+</div>
+<section id="about-me">
+    <h2>About Me</h2>
+    <p>I am a software developer with a deep passion for understanding how technology can help people and enable me to create amazing apps as well as software. I'm currently training at Mayden Academy while learning the following languages: HTML, CSS, Js and PHP. I'm on week 2 and can't wait to learn more.</p>
+    <p>When not coding my hobbies include sports such as basketball, bouldering and longboarding, photography, travelling, learning languages (mainly Korean) and exploring museums, cafes / restaurants, exhibitions.</p>
+</section>
+<main>
+    <section>
+        <h2>Projects</h2>
+        <section>
+            <section id="projects">
+                <div class="project_image">
+                    <a href="https://codepen.io/sobiaf/pens/"><h3>Answer Game</h3>
+                        <img alt="project 1 answer game" src="file:///Users/sobia/Desktop/for%20portfolio/Screenshot%202021-03-09%20at%2016.20.51.png"></a>
+                </div>
+                <div class="project_image">
+                    <a href= https://github.com/SobiaF/simple-calculator-using-java> <h3>Calculator</h3>
+                        <img src="file:///Users/sobia/Desktop/for%20portfolio/photo-1612622826776-cd8b6da23632.jpeg" alt="calculator"></a>
+                </div>
+                <div class="project_image">
+                    <a href= https://github.com/SobiaF/continents-and-cities>
+                        <h3>Continents and Cities</h3>
+                        <img src="file:///Users/sobia/Desktop/for%20portfolio/Paris%20-%20by%20SF.jpg" alt="Paris image with partial Eiffel Tower"></a>
+                </div>
+                <div class="project_image">
+                    <a href= https://github.com/SobiaF/simple-car-loan-payment-calculator> <h3>Car Loan Payment Calculator</h3>
+                        <img src="file:///Users/sobia/Desktop/for%20portfolio/photo-1471174617910-3e9c04f58ff5.jpeg" alt="steering wheel car interior"></a>
+                </div>
+                <div class="project_image">
+                    <a href= https://github.com/SobiaF/petsupplyshop><h3>Pet (food) Supply Shop</h3>
+                        <img src="file:///Users/sobia/Desktop/for%20portfolio/photo-1571151596869-2663a84c1e41.jpeg" alt="cat on cardboard box with advert"></a>
+                </div>
+                <div class="project_image">
+                    <a href= https://github.com/SobiaF/simple-magic-maths-trick><h3>Magic Math Trick</h3>
+                        <img src="file:///Users/sobia/Desktop/for%20portfolio/photo-1518133910546-b6c2fb7d79e3.jpeg" alt="pencil and paper with maths"></a>
```

Comment:
I'm not able to see these images, could you add them to the repo and push them up?

Also, could you update the image src for these images? the way they are set up means that they will only ever work on your mac. What you can do is make sure all of the images are in the same folder as this index.html file, and then the src attribute only needs to contain the name of the image file eg

`<img src="photo-1518133910546-b6c2fb7d79e3.jpeg">`